docs/Release Notes.md

```diff
@@ -0,0 +1,17 @@
+# Release Notes
+Release notes are an important part of the release process and we aim to provide high quality notes with our software, by:
+1. _Writing the notes collaboratively_, because nobody knows how to describe a feature to the users better than the team that worked on it.
+2. _translating them_, so that more people are able to get them.
+
+
+## Release notes workflow
+1. In the root folder of every project, we maintain a `RELEASE-NOTES.txt` file that is updated by every PR that adds a relevant feature. As a reminder of developers and reviewers, we usually add a note about this in the project GitHub PR template. 
+2. The `get_prs_list_action` action in the `release-toolkit`  provides a list of all the merged PRs  in a range of tags. 
+3. During the code freeze, the release manager gets the release notes from `1` , uses `2` to get the list of merged PRs to verify/integrate the notes and write down the final copy.
```

Comment:
It _might_ be more understandable to use "`Step 1`" rather than just `1`, but 🤷‍♂